src/borg/archiver/lock_cmds.py

```diff
@@ -73,7 +73,11 @@ def build_parser_locks(self, subparsers, common_parser, mid_common_parser):
 
         with_lock_epilog = process_epilog(
             """
-        This command runs a user-specified command while the repository lock is held.
+        This command runs a user-specified command while locking the repository. For example:
+
+        ::
+
+        $ borg with-lock /mnt/borgrepo rsync -av /mnt/borgrepo /somewhere/else
```

Comment:
please indent line 80 by 4 spaces.

also, guess it should be `/somwhere/else/borgrepo`.